db/dataset_eval.py

```diff
@@ -11,12 +11,14 @@
 STATUS_RUNNING = "running"
 STATUS_DONE = "done"
 STATUS_FAILED = "failed"
-
+# Eval locations are defined in `eval_location_type` type. See schema definition.
+EVAL_LOCAL = "local"
+EVAL_REMOTE = "remote"
 # Filter types are defined in `eval_filter_type` type. See schema definition.
 FILTER_ARTIST = "artist"
 
 
-def evaluate_dataset(dataset_id, normalize, filter_type=None):
+def evaluate_dataset(dataset_id, normalize, filter_type=None, eval_location=EVAL_LOCAL):
```

Comment:
There are two ways of making sure that we don't miss code from other places:
 * use grep to check if the method is called anywhere else
 * use our tests

Ideally we should be using tests, but I know that we don't always have tests covering all of the website. You should always be aiming to make the "correct" change, instead of the "easiest", even if it means that you need to go to some more effort to change other parts of the code.